src/scripts/Deploy-Toolkit.ps1

```diff
@@ -45,9 +46,19 @@ If ($null -eq $rg) {
 }
 
 # Start deployment
-$params = @{ hubName = $ResourceGroup };
-New-AzResourceGroupDeployment `
+$params = @{ 
+    hubName = $ResourceGroup
+    ExportScopes = $ExportScopes
+};
+$hub = New-AzResourceGroupDeployment `
     -TemplateFile "../templates/$Template/main.bicep" `
     -TemplateParameterObject $params `
     -ResourceGroupName $ResourceGroup `
     -WhatIf:$WhatIf
+
+Start-AzDataFactoryV2Trigger `
```

Comment:
Don't we need to wait for the deployment to be complete before triggering this? Can we move it to bicep and do it as part of the main deployment?

Reply:
For some reason I cannot fathom one can't start the trigger from the deployment or have it default to enabled.  I have the same challenge in my "full" solution

Reply:
Have you asked this on the WW Azure group? We should get an official response from the ADF team.

Reply:
(This doesn't have to be blocking. We should probably track it with an issue and a TODO in code.)